src/ONTraC/GNN/_GNN.py

```diff
@@ -146,6 +147,7 @@ def predict(output_dir: str, batch_train: SubBatchTrainProtocol, dataset: Spatai
         # consolidate out_adj
         ind = torch.arange(consolidate_s.shape[-1], device=consolidate_out_adj.device)  # type: ignore
         consolidate_out_adj[ind, ind] = 0  # type: ignore
+        consolidate_out_adj = out_adj_norm(consolidate_s, consolidate_out_adj)
         d = torch.einsum('ij->i', consolidate_out_adj)
         d = torch.sqrt(d)[:, None] + 1e-15
         consolidate_out_adj = (consolidate_out_adj / d) / d.transpose(0, 1)
```

Comment:
1)Does we still need these steps after new normalization?

Reply:
2) We could make the different normalization methods as choice in the utils functions.